src/Raku/Actions.nqp

```diff
@@ -2522,14 +2522,14 @@ class Raku::Actions is HLL::Actions does Raku::CommonActions {
 
         my $decl;
         my $shape := $<semilist> ?? $<semilist>[0].ast !! Nodify('SemiList');
-        if $<desigilname> {
-            my $desigilname := $<desigilname>;
+        if $<desigilname> || $<variable><desigilname> {
```

Comment:
Isn't it always `$<variable><desigilname>` now? `variable-declarator` doesn't have a desigilname submatch anymore.

Reply:
> Isn't it always `$<variable><desigilname>` now? `variable-declarator` doesn't have a desigilname submatch anymore.

Yep, thanks for catching that.